WordPress/Classes/ViewRelated/NUX/ContentViews/UnifiedPrologueEditorContentView.swift

```diff
@@ -0,0 +1,87 @@
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+
+struct UnifiedPrologueEditorView: View {
+
+    var body: some View {
+
+        VStack {
+            Spacer()
+            ZStack {
+                RoundedRectangle(cornerRadius: 4, style: .continuous)
+                    .foregroundColor(Appearance.textBackgroundColor)
+                Text(Appearance.topElementTitle)
+                    .font(Appearance.largeTextFont)
+            }
+            .fixedSize(horizontal: false, vertical: true)
+            Spacer(minLength: 16)
+                .fixedSize(horizontal: false, vertical: true)
+            ZStack {
+                RoundedRectangle(cornerRadius: 4, style: .continuous)
+                    .foregroundColor(Appearance.textBackgroundColor)
+                (Text(Appearance.middleElementTitle)
+                    + Text(Appearance.middleElementTerminator)
+                        .foregroundColor(.blue))
+                    .lineLimit(.none)
+                    .padding()
+
+            }
+            .fixedSize(horizontal: false, vertical: true)
+            Spacer(minLength: 16)
+                .fixedSize(horizontal: false, vertical: true)
+            ZStack {
+                RoundedRectangle(cornerRadius: 4, style: .continuous)
+                    .foregroundColor(Appearance.textBackgroundColor)
+                HStack {
+                    Image("page2Img1Sea")
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                    Image("page2Img2Trees")
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                    Image("page2Img3Food")
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                }
+                .padding(.all, 8)
+            }
+            .fixedSize(horizontal: false, vertical: true)
+            Spacer()
+        }
+    }
+}
+
+private extension UnifiedPrologueEditorView {
+
+    enum Appearance {
+        static let topElementTitle: LocalizedStringKey = "Getting Inspired"
+        static let middleElementShortTitle: LocalizedStringKey = "I am so inspired by photographer Cameron Karsten's work. I will be trying these techniques on my"
+        static let middleElementTerminator = "|"
+        static let textBackgroundColor = Color(UIColor(light: .white, dark: .gray(.shade90)))
+        static let largeTextFont = Font(WPStyleGuide.serifFontForTextStyle(.title1))
+        static let normalTextFont = Font.subheadline
+        /// - TODO: This needs to be updated with actual text
+        static let middleElementLongTitle: LocalizedStringKey = "I am so inspired by photographer Cameron Karsten's work. I will be trying these techniques on my I am so inspired by photographer Cameron Karsten's work. I will be trying these techniques on my I am so inspired by photographer Cameron Karsten's work. I will be trying these techniques on my"
```

Comment:
Can we jsut scale things up instead of adding more text?

Reply:
yes. in general, we will revisit iPad layout based on the conversations we had on Slack